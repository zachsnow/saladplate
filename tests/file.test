A simple test of file inclusion: $<< file.incl >>
