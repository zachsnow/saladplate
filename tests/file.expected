A simple test of file inclusion: A simple include file.
